gdx/src/com/badlogic/gdx/utils/Align.java

```diff
@@ -30,6 +30,8 @@ public class Align {
 	static public final int bottomLeft = bottom | left;
 	static public final int bottomRight = bottom | right;
 
+	private static String separator = ",";
```

Comment:
I don't think it's necessary to make the separator for the `toString`-method configurable...